pydicom/dataset.py

```diff
@@ -2014,6 +2044,11 @@ def to_json(self, bulk_data_threshold=1, bulk_data_element_handler=None,
             serialized (dumped) JSON string (by default uses
             :func:`json.dumps`).
 
+            .. note:
+
+                Make sure to use a dump handler that sorts the keys (see
+                example below) do create DICOM-conform JSON.
```

Comment:
do -> to

Reply:
Should have done a self-review first - I see this kind of errors only if someone else does it ;)

Reply:
Me too